src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -35,6 +32,9 @@ private sealed class State
             // It must be kept alive until we receive SHUTDOWN_COMPLETE event
             public GCHandle StateGCHandle;
 
+            // Backing for StreamId
+            public long _streamId = -1;
```

Comment:
Nit for consistency:
```suggestion
            public long StreamId = -1;
```